### PR TITLE
Rs sync warn

### PR DESCRIPTION
**Pull request by AdrianLxM**
Warns when the time is synchronized on Dana RS or Rv2 and the time difference was more than 1.5 hours.

This should not disturb on standard DST changes but at least warns the user.
It would still be good to be able to automatically sync the events (if we could get the info off the pump) or erase the pumps "history" that is wrong (missing the commands?). But at least the user is warned when he travels through time zones.

Thanks @JustinWalkerMDPL for reporting this.

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -1172,6 +1172,8 @@
     <string name="combo_invalid_setup">Invalid pump setup, check the docs and verify that the Quick Info menu is named "QUICK INFO" using the 360 configuration software.</string>
     <string name="custom">Custom</string>
     <string name="key_lockscreen" translatable="false">lockscreen</string>
+    <string name="largetimedifftitle">Large Time Difference</string>
+    <string name="largetimediff">Large time difference: Time in pump was adjuted for more than 1.5 hours. Make sure that reading the history from the pump does not cause unexpected behaviour.</string>
```
Comment (MilosKozak) [inline app/src/main/res/values/strings.xml]: typo "adjuted"

**Comment by AdrianLxM** [comment]
@MilosKozak what do you think about the solution to NOT sync in that case (long time diff) but warn the user to do it manually?

**Comment by MilosKozak** [comment]
this warning comes too late. clean solution would be keep the pump uninitialized until you do it

**Comment by MilosKozak** [comment]
@AdrianLxM

**Comment by AdrianLxM** [comment]
There is still a problem re-initializing the pump after a correct time-sync if the user adjusted the time manually :/

**Comment by AdrianLxM** [comment]
now it gets re-initialized again.

**Comment by AdrianLxM** [comment]
Thank you @Heiner1 for testing the Rv2 part!

**Comment by MilosKozak** [comment]
sounds good to me ne now. thanks guys!

**Pull request merged**